django/contrib/syndication/views.py

```diff
@@ -134,7 +135,7 @@ def get_feed(self, obj, request):
             subtitle=self._get_dynamic_attr('subtitle', obj),
             link=link,
             description=self._get_dynamic_attr('description', obj),
-            language=settings.LANGUAGE_CODE,
+            language=self.language or get_language(),
```

Comment:
This would be backwards compatible in the sense that it select the active language and not the one that is the default in the settings right? which is probably okay since the feedparsers are most likely not the users browser anyways and have their own session

Reply:
> which is probably okay since the feedparsers are most likely not the users browser anyways and have their own session

This was my reasoning, too. It's not completely backwards compatible, but I figured that up until now, probably nobody worked with the language tag anyways, since they couldn't change it (without adjusting settings at least temporarily). But I don't have strong feelings either way.

Reply:
Please add a note in the 3.0 release notes (`Backwards incompatible changes in 3.0 -> Miscellaneous`).